ros2topic/ros2topic/verb/pub.py

```diff
@@ -56,6 +58,20 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '-n', '--node-name', type=str,
             help='Name of the created publishing node')
+        parser.add_argument(
+            '--qos-profile', type=str,
+            choices=QOS_PROFILE_OPT.keys(),
+            help='Quality of service profile to publish with')
```

Comment:
Just an idea (doesn't have be done): it would be neat if the help message would print the details about every profile so the users sees what they stand for :bulb: